man/man4/spl.4

```diff
@@ -65,9 +65,10 @@ Objects of size
 .Sy spl_kmem_cache_slab_limit
 or smaller will be allocated using the Linux slab allocator,
 large objects use the SPL allocator.
-A cutoff of 16K was determined to be optimal for architectures using 4K pages.
+A cutoff of 16 KiB was determined to be optimal for architectures using 4 KiB
+pages.
 .
-.It Sy spl_kmem_alloc_warn Ns = Ns Sy 32768 Pq uint
+.It Sy spl_kmem_alloc_warn Ns = Ns Sy 65536 Pq uint
```

Comment:
The correction of the value should be its own commit, rather than ninja included in this one.